internal/core/context.rs

```diff
@@ -74,3 +77,12 @@ pub fn with_platform<R>(
         }
     })
 }
+
+pub fn set_window_shown_hook(
```

Comment:
could you add documentation comment for when the hook is shown?

Reply:
Can do :)